src/main/java/se/sundsvall/supportcenter/service/mapper/constant/CaseMapperConstants.java

```diff
@@ -89,7 +89,7 @@ private CaseMapperConstants() {}
 	private static final String IT_SUPPORT = "IT Support";
 
 	public static final Map<String, List<CustomStatusMapping>> CUSTOM_STATUS_MAP = Stream.concat(getCubeStatusMap().entrySet().stream(),
-		getNetsetStatusMap().entrySet().stream()).collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue));
+		getNetsetStatusMap().entrySet().stream()).collect(Collectors.toUnmodifiableMap(Map.Entry::getKey, Map.Entry::getValue));
```

Comment:
No need to add more entries to this map?

Reply:
No